### PR TITLE
feat(container): update link support to Snow for EU/CA

### PR DESCRIPTION
**Pull request by darsene**
ref: MANAGER-13069

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/helpcenter-GA-migration-to-SNOW`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-13069
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~Only FR translations have been updated~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~Breaking change is mentioned in relevant commits~

## Description

Migrate support links to Snow for EU/CA region

### CODE REVIEW

File: packages/manager/apps/container/src/container/legacy/server-sidebar/SidebarVirtual.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export default function ServerSidebarVirtual({
       } else {
         item.isOpen = !item.isOpen;
       }
-      if (item.href) {
+      if (item.href && !item.isExternal) {
```
Comment (frenautvh) [inline packages/manager/apps/container/src/container/legacy/server-sidebar/SidebarVirtual.tsx:39]: what about all the items that have an href and are not flagged external ? won't this the navigation for a lot of items in the sidebar ?

Comment (darsene) [reply packages/manager/apps/container/src/container/legacy/server-sidebar/SidebarVirtual.tsx:39]: For non external links, the behavior is equal to currently, nothing changed. There is a bug for external links, even tough a new tab is opened, the link is also loaded inside the manager app. 
see `window.location.href = item.href`

**Review by chipp972 — APPROVED** [review]

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/Assistance.tsx [hunk 2]
```diff
@@ -80,6 +80,17 @@ const AssistanceSidebar: React.FC<ComponentProps<Props>> = ({
     trackingPlugin.trackClick({ name: `navbar_v2_${id}`, type: 'navigation' });
   };
 
+  const supportNode = {
```
Comment (qpavy) [inline packages/manager/apps/container/src/container/nav-reshuffle/sidebar/Assistance.tsx:83]: Don't we have a type or an interface for this object ?

**Review by frenautvh — APPROVED** [review]

File: packages/manager/apps/container/src/container/legacy/server-sidebar/universe/AccountSidebar.tsx [hunk 3]
```diff
@@ -31,6 +32,8 @@ export default function AccountSidebar() {
 
     const menu = [];
 
+    const isEUCA = ['EU', 'CA'].includes(region);
```
Comment (anooparveti) [inline packages/manager/apps/container/src/container/legacy/server-sidebar/universe/AccountSidebar.tsx:35]:
```suggestion
    const isEUOrCA = ['EU', 'CA'].includes(region);
```

File: packages/manager/apps/container/src/container/nav-reshuffle/sidebar/Assistance.tsx [hunk 4]
```diff
@@ -33,7 +33,7 @@ const AssistanceSidebar: React.FC<ComponentProps<Props>> = ({
   const reketInstance = useReket();
   const [selectedItem, setSelectedItem] = useState<string>(null);
 
-  const hasAdvancedSupport = ['EU', 'CA'].includes(environment.getRegion());
+  const isEUCA = ['EU', 'CA'].includes(environment.getRegion());
```
Comment (anooparveti) [inline packages/manager/apps/container/src/container/nav-reshuffle/sidebar/Assistance.tsx:36]:
```suggestion
  const isEUOrCA = ['EU', 'CA'].includes(environment.getRegion());
```

File: packages/manager/apps/container/src/container/legacy/account-sidebar/UsefulLinks/useUsefulLinks.ts [hunk 5]
```diff
@@ -19,6 +19,8 @@ const useUsefulLinks = (): UseUsefulLinks => {
   const user = environment.getUser();
   const { isLivechatEnabled, setChatbotReduced } = useContainer();
 
+  const isEUCA = ['EU', 'CA'].includes(region);
```
Comment (anooparveti) [inline packages/manager/apps/container/src/container/legacy/account-sidebar/UsefulLinks/useUsefulLinks.ts:22]:
```suggestion
  const isEUOrCA = ['EU', 'CA'].includes(region);
```

**Review by anooparveti — APPROVED** [review]

**Pull request merged**